infrastructure/code/backend_service.go

```diff
@@ -90,7 +91,12 @@ func NewHTTPRepository(
 	return &SnykCodeHTTPClient{client, instrumentor, errorReporter}
 }
 
-func (s *SnykCodeHTTPClient) GetFilters(ctx context.Context) (configFiles []string, extensions []string, err error) {
+func (s *SnykCodeHTTPClient) GetFilters(ctx context.Context) (
+	configFiles []string,
```

Comment:
You can replace multiple return types with a single `filterResponse` type to have cleaner interface.

Reply:
✅ Done, very nice point! I used you suggestion to replace it with `filterResponse` -- let me know if you would like to have a mirroring struct not in the `http` world.

Reply:
No need for a mirroring struct, we won't win from it, let's keep it simple.